src/Autoloader.php

```diff
@@ -54,25 +59,39 @@ public static function load()
         ));
     }
 
-    /**
-     * @return ClassLoader
-     * @throws RuntimeException
-     */
-    private static function getClassLoader()
+    private static function getClassLoader(): ?ClassLoader
     {
         if (getenv('COMPOSER_VENDOR_DIR') && file_exists(getenv('COMPOSER_VENDOR_DIR') . '/autoload.php')) {
```

Comment:
Can we get rid of the repeated `getenv` calls here?

Reply:
Fixed.